k8s/operators/src/pool/error.rs

```diff
@@ -43,6 +43,10 @@ pub enum Error {
     Generic {
         message: String,
     },
+    #[snafu(display("{}", message))]
```

Comment:
This seems quite generic, and we already have a generic error, so why not add display to that one instead?

Reply:
I am using this for CRD operation errors.

Reply:
Shall i use generic itself?